### PR TITLE
Feat/popupmenu

### PR DESCRIPTION
**Pull request by mmehmetAliIzci**
Add popupmenu styles

### CODE REVIEW

File: styles/native/js/core/widgets/popupmenu.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { font } from "../variables";
+import { brand } from "../../app/custom-variables";
```
Comment (wegiswes) [inline styles/native/js/core/widgets/popupmenu.js]: brand has to be imported from `../variables`

File: styles/native/ts/core/widgets/popupmenu.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { font }        from "../variables";
+import { PopupMenuType } from "../../types/widgets";
+import { brand } from "../../app/custom-variables";
```
Comment (wegiswes) [inline styles/native/ts/core/widgets/popupmenu.ts]: brand has to be imported from `../variables`

File: styles/native/ts/core/widgets/popupmenu.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+		shadowColor: "#000",
+        shadowOpacity: 0.2,
+        shadowRadius: 10,
+        elevation: 16,
+	},
+	basic: {
+        dividerColor: font.color,
+		itemStyle: {
+			defaultStyle: {
+                // All TextStyle properties are allowed
```
Comment (wegiswes) [inline styles/native/ts/core/widgets/popupmenu.ts:29]: fix indents please 😄

File: styles/native/ts/core/widgets/popupmenu.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        // All ViewStyle properties are allowed
+		borderRadius: 10,
+		shadowColor: "#000",
+        shadowOpacity: 0.2,
+        shadowRadius: 10,
+        elevation: 16,
+	},
+	basic: {
+        dividerColor: font.color,
+		itemStyle: {
```
Comment (wegiswes) [inline styles/native/ts/core/widgets/popupmenu.ts]: missing ellipsizeMode

**Review by wegiswes — CHANGES_REQUESTED** [review]

File: styles/native/ts/core/widgets/popupmenu.ts (hunk 4 above)
Comment (mmehmetAliIzci) [reply styles/native/ts/core/widgets/popupmenu.ts]: Done

**Review by wegiswes — APPROVED** [review]

**Pull request merged**